README.md

```diff
@@ -200,7 +225,14 @@ returned with `usage` and `limit` set to `0`.
 }
 ```
 
-### `report(org, feature, [n = 1], [at = new Date()], [clobber = false])`
+### `report(org, feature, [n = 1])`
+
+Report usage of a feature by an org.
+
+The optional `n` parameter indicates the number of units of the
+feature that were consumed.
+
+### `reportUsage({ org, feature, n = 1, at = new Date(), clobber = false })`
```

Comment:
If `clobber` is `true`, `n` should be explicitly set; otherwise, one can clobber to 0 inadvertently.

Reply:
Yeah, this method was removed anyway.  I can add a check to ensure that `n` is a number, but it's pretty tricky to do by accident with `report(org, feature, n, {options})`, especially since TS will complain about it if you do `report(org, feature, undefined, {options})`